cms/serializers.py

```diff
@@ -2,6 +2,12 @@
 from cms import models
 
 
+class MenuItemShortSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.MenuItem
+        exclude = ['priority', 'sites']
```

Comment:
Preco nechceme zobrazovat prioritu a sites v API? Nebudeme to potrebovať na poradie menu itemov?

Reply:
už ti prídu zoradené, tak mi to prišlo ako zbytočnosť. Či ieš si predstaviť že to niekde použiješ?

Reply:
Praveze na tom endpointe su zoradene podla id.